### PR TITLE
Removing var openshift_logging_es5_techpreview

### PR DESCRIPTION
**Pull request by ewolinetz**
This shouldn't go in until master is open for 3.11

As part of 3.11 we will be moving to deploying the ES 5.x stack (it will no longer be tech preview)

### CODE REVIEW

**Comment by ewolinetz** [comment]
Squashed and removed [WIP] tag since we branched for 3.11.
May be required to go in before https://github.com/openshift/origin-aggregated-logging/pull/1214 can be tested properly

**Comment by ewolinetz** [comment]
kibana wasn't able to start up in time...
retesting in case its just a flake
/test logging

**Comment by ewolinetz** [comment]
failed while testing utf8-characters.sh
looks like a flake (the prior command passed)
/retest

**Comment by ewolinetz** [comment]
Looks like the logging ci test is building our component images as `origin-logging-{component}:latest` instead of `origin-logging-{component}5:latest` where necessary

**Comment by ewolinetz** [comment]
possible flakes with fluentd catching up
/test logging

**Comment by ewolinetz** [comment]
failed while waiting for fluentd to catch up again...
checked on the machine and that query returned 0 entries against the es-ops cluster
```
[root@ip-172-18-12-146 ec2-user]# oc exec -c elasticsearch logging-es-ops-data-master-ma68hcen-1-sffcz -- es_util --query=/.operations.*/_count -X POST -d '{"query":{"bool":{"filter":{"match_phrase":{"message":"GET /a2d325e7c5b5458e9e75301d459b8757 404 "}},"must":{"term":{"kubernetes.container_name":"kibana"}}}}}'
{"count":0,"_shards":{"total":6,"successful":6,"skipped":0,"failed":0}}

[root@ip-172-18-12-146 ec2-user]# oc exec -c elasticsearch logging-es-ops-data-master-ma68hcen-1-sffcz -- es_util --query=_cat/indices?v
health status index                                            uuid                   pri rep docs.count docs.deleted store.size pri.store.size
green  open   .kibana.d033e22ae348aeb5660fc2140aec35850c4da997 yfRRQfIORXKFwmnT6nfQqw   1   0          6            1     96.7kb         96.7kb
green  open   .kibana.9c894d6e0d2a5bde27f0350f4416a48c9c9c570b nN2CBc7WToe7GQXJBRLrSw   1   0          1            1     26.2kb         26.2kb
green  open   .operations.2018.06.20                           wl4Fnc4HSgSPRIBiMJKNVw   1   0     662634            0    438.8mb        438.8mb
yellow open   .kibana                                          C8R7pydCSOKVy4Nl6Nb9Yg   1   1          1            0      3.2kb          3.2kb
green  open   .kibana.53f493bd77bfee804e7c2fae9af0aa104dad4a5d glKk-1WJSN-hUNs5R4P1Nw   1   0          1            1     26.2kb         26.2kb
green  open   .searchguard.logging-es-ops-data-master-ma68hcen NlZ4rMQRTSagrimc_cdyUA   1   0          5            0     31.5kb         31.5kb

[root@ip-172-18-12-146 ec2-user]# oc exec -c elasticsearch logging-es-ops-data-master-ma68hcen-1-sffcz -- es_cluster_health
{
  "cluster_name" : "logging-es-ops",
  "status" : "yellow",
  "timed_out" : false,
  "number_of_nodes" : 1,
  "number_of_data_nodes" : 1,
  "active_primary_shards" : 6,
  "active_shards" : 6,
  "relocating_shards" : 0,
  "initializing_shards" : 0,
  "unassigned_shards" : 1,
  "delayed_unassigned_shards" : 0,
  "number_of_pending_tasks" : 0,
  "number_of_in_flight_fetch" : 0,
  "task_max_waiting_in_queue_millis" : 0,
  "active_shards_percent_as_number" : 85.71428571428571
}
```

**Comment by ewolinetz** [comment]
dependent on https://github.com/openshift/origin-aggregated-logging/pull/1217 for tests

**Comment by richm** [comment]
now down to 2 failures: multi_tenancy.sh and viaq - I have a fix for viaq - not sure about multi_tenancy.sh

**Comment by ewolinetz** [comment]
https://github.com/openshift/origin-aggregated-logging/pull/1220 opened to address multitenancy

**Comment by sdodson** [comment]
I think it's using top level OWNERS because there's no OWNERS defined in roles/openshift_logging_defaults/

File: playbooks/openshift-logging/private/config.yml [hunk 1]
```diff
@@ -17,33 +17,29 @@
   hosts: all
```
Comment (deads2k) [inline playbooks/openshift-logging/private/config.yml]: This should be restricted to the nodes running logging (infra), right?

**Comment by deads2k** [comment]
@liggitt @derekwaynecarr @jwforres Looks like 3.10 didn't have this, so we don't have to worry about the upgrade case.

@derekwaynecarr is this the way to do this or should it be a special kind of pod instead?

**Pull request merged**